tests.py

```diff
@@ -228,9 +278,10 @@ def doctest_edge_label_long_type_names():
     """
 
 
-def test_suite():
+def suite():
```

Comment:
Why rename this?  Please don't.  The name of this function is [referenced from setup.py](https://github.com/mgedmin/objgraph/blob/95017a8c37dc1e6f3f70a27fd675eefb650add6b/setup.py#L11).